worker/worker.go

```diff
@@ -1037,6 +1090,7 @@ func (w *worker) Handler() http.Handler {
 		"GET    /account/:hostkey": w.accountHandlerGET,
 		"GET    /id":               w.idHandlerGET,
 
+		"POST   /rhp/broadcast/:id":   w.rhpBroadcastHandler,
```

Comment:
I think `/rhp/contract/:id/broadcast` is better

Reply:
I feel like `rhp` would be a bit much if we also want to scope it to contracts. Maybe `/contract/:id/broadcast`?
The `rhp` segment is not really meaningful imo and if we redid it from scratch I'd probably also scope the contract formation and renewal under `/contract/`.